src/redist/targets/BundledManifests.targets

```diff
@@ -0,0 +1,72 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <BundledManifests Include="Microsoft.NET.Workload.Android" Version="$(XamarinAndroidWorkloadManifestVersion)" WorkloadId="Microsoft.NET.Workload.Android" />
```

Comment:
Workloads and workload manifests don't have a 1 to 1 relationship.  Both the iOS and Android workloads could be in a "Mobile" manifest, for example.

So I would use WorkloadManifestId instead of WorkloadId here.